service/src/test/java/uk/gov/hmcts/reform/fpl/model/CaseDataTest.java

```diff
@@ -1208,6 +1210,134 @@ void shouldReturnEmptyWhenNoHearingOrdersBundlesExistForApproval() {
         }
     }
 
+    @Nested
+    class GetHearingOrderDraftCMOs {
+        @Test
+        void shouldReturnAListOfDraftCaseManagementOrders() {
+            Element<HearingOrder> draftCMOOne = element(randomUUID(), buildHearingOrder(DRAFT_CMO));
+            Element<HearingOrder> draftCMOTwo = element(randomUUID(), buildHearingOrder(AGREED_CMO));
+            Element<HearingOrder> draftCMOThree = element(randomUUID(), buildHearingOrder(DRAFT_CMO));
+
+            CaseData caseData = CaseData.builder()
+                .hearingOrdersBundlesDrafts(List.of(
+                    element(HearingOrdersBundle.builder().orders(List.of(draftCMOOne, draftCMOTwo)).build()),
+                    element(HearingOrdersBundle.builder().orders(List.of(
+                        draftCMOThree,
+                        element(buildHearingOrder(C21))
+                    )).build())))
+                .build();
+
+            assertThat(caseData.getHearingOrderDraftCMOs()).isEqualTo(List.of(draftCMOOne, draftCMOTwo, draftCMOThree));
+        }
+
+        @Test
+        void shouldReturnAnEmptyListIfNoCaseManagementOrdersHaveBeenMade() {
+            CaseData caseData = CaseData.builder()
+                .hearingOrdersBundlesDrafts(List.of(
+                    element(HearingOrdersBundle.builder().orders(List.of(
+                        element(buildHearingOrder(C21)))).build()),
+                    element(HearingOrdersBundle.builder().orders(List.of(
+                        element(buildHearingOrder(C21))
+                    )).build())))
+                .build();
+
+            assertThat(caseData.getHearingOrderDraftCMOs()).isEmpty();
+        }
+
+        @Test
+        void shouldReturnAnEmptyListIfHearingOrderBundlesDoNotExist() {
+            CaseData caseData = CaseData.builder().build();
+
+            assertThat(caseData.getHearingOrderDraftCMOs()).isEmpty();
+        }
+    }
+
+    @Nested
+    class GetDraftUploadedCMOWithId {
+        @Test
+        void shouldReturnHearingOrderBundleWhenOrdersContainOrderId() {
```

Comment:
Test name is not right. Test mentions `HearingOrderBundle`but test is about `draftUploadedCMOs`